README.md

```diff
@@ -17,9 +17,9 @@ npm install -g tsuml
 ## Usage
 
 ```
-tsuml --glob ./src/**/*.ts
+tsuml --glob './src/**/*.ts'
```

Comment:
This same change needs to be made in `src/bin/index.ts` in the line `console.log(chalk.yellowBright("tsuml --glob ./src/**/*.ts"));`